tests/Unit/Models/TraderModelTest.php

```diff
@@ -40,7 +43,19 @@ public function testSoftDeleteTrader()
         $this->assertCount(0, Trader::all());
     }
 
-    public function testTraderBelongsToMarket() {
+    public function testItCanBeDisabledAndEnabled()
+    {
+        $this->trader->disable();
+        $this->trader->refresh();
+        $this->assertInstanceOf(Carbon::class, $this->trader->disabled_at);
```

Comment:
maybe also a `$this->assertNotNull`? might be redundant though :-)

Reply:
I think redundant, as null would not be an instance of class